### PR TITLE
Refactored database to postgres

### PR DESCRIPTION
**Pull request by MichaelDimitras**
@lhmun This PR is just me switching from mongo to postgres.  I implemented a new database file in postgres.js and changed a few lines in server.js to work with the new database.

### CODE REVIEW

File: database/postgres.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const config = {
+  user: process.env.USER || 'postgres',
+  host: 'localhost',
+  database: 'muncheez',
+  password: null,
+  port: process.env.PORT || 5432,
+};
+
+const pool = new Pool(config);
+console.log(pool);
```
Comment (louismun) [inline database/postgres.js]: Consider removing this console log

File: database/postgres.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const getRestaurant = function (id, cb) {
+  pool.query(`SELECT * FROM restaurants WHERE id=${id}`, (err, data) => {
+    if (err) {
+      console.log(err);
+    }
+    console.log(data.rows[0]);
```
Comment (louismun) [inline database/postgres.js]: Here too

File: server/server.js [hunk 3]
```diff
@@ -24,13 +24,14 @@ app.get('/restaurants/:id', (req, res) => {
...
 app.get('/api/restaurants/:id', (req, res) => {
-  const q = Places.findOne({ id: req.params.id });
-
-  q.exec((err, place) => {
-    if (err) { console.log(err); }
-    console.log('PLACE: ', place);
-    res.send(place);
-  });
+  Places.getRestaurant(req.params.id, res.send.bind(res));
+  // const q = Places.findOne({ id: req.params.id });
```
Comment (louismun) [inline server/server.js]: and here

**Review by louismun — CHANGES_REQUESTED** [review]
Just remove comments/console logs

**Comment by MichaelDimitras** [comment]
@lhmun I made the changes you asked for.  Thanks for saying something about the console.logs. It was sloppy for me to leave those in. I want to leave the commented out code for the mongo query for the time being in case I end up switching back

**Review by louismun — APPROVED** [review]

**Pull request merged**